### PR TITLE
Replaced Management API section with link to QS

### PR DESCRIPTION
**Pull request by cocojoe**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -274,55 +274,11 @@ Auth0
     }
 ```
 
-### Management API (Users)
+### Management API
 
-#### Retrieve user_metadata
+You can request more information than returned in the basic profile and manage the user's metadata. To do this, you can access the Auth0 [Management API](https://auth0.com/docs/api/management/v2)
```
Comment (lbalmaceda) [inline README.md]: "than returned" ? sounds odd. Can you rephrase that?

File: README.md [hunk 2]
```diff
@@ -274,55 +274,11 @@ Auth0
     }
 ```
 
-### Management API (Users)
```
Comment (lbalmaceda) [inline README.md]: This was here because the MGMT API implemented on these mobile sdks is only meant to do small "User's" stuff, such as get full profile or link accounts. If you remove it from here you might want to add this clarification somewhere below.

File: README.md [hunk 3]
```diff
@@ -274,55 +274,11 @@ Auth0
...
-        case .success(let userInfo):
-            print("User: \(userInfo)")
-        case .failure(let error):
-            print("Failed with \(error)")
-        }
-    }
-```
-
-#### Link an account
-
```
Comment (lbalmaceda) [inline README.md]: No need to remove this section

File: README.md (hunk 1 above)
Comment (cocojoe) [reply README.md]: than is used when you're talking about comparisons. Also a cp from QS.

File: README.md (hunk 3 above)
Comment (cocojoe) [reply README.md]: Agreed, accidental.
